### PR TITLE
Re-starting the consumer in case of network failer or kafka brokers a…

### PR DESCRIPTION
**Pull request by sharadgaur**
I am restarting the consumer in case Kafka consumer was not able to recover/re-connect/re-consume in case of network outage between consumer and broker or kafka brokers got restarted.

Also added the new config variable retry_wait_seconds (default value 30) for delay.

~Sharad

### CODE REVIEW

**Comment by sharadgaur** [comment]
@repeatedly Please have a look and let me know if I need to change something. We need this change in production asap.

Thanks,
Sharad

File: lib/fluent/plugin/in_kafka_group.rb [hunk 1]
```diff
@@ -176,6 +191,7 @@ def run
             rescue => e
               log.warn "parser error in #{batch.topic}/#{batch.partition}", :error => e.to_s, :value => msg.value, :offset => msg.offset
               log.debug_backtrace
+              reconnect_consumer
```
Comment (repeatedly) [inline lib/fluent/plugin/in_kafka_group.rb]: Is this point correct for reconnecting?
This `rescue` is for catching parsing error so kafka brokers should not happen this point.

Comment (sharadgaur) [reply lib/fluent/plugin/in_kafka_group.rb]: oh good catch .. let me fix and re-test it

**Comment by sharadgaur** [comment]
@repeatedly  Please review.

**Comment by sharadgaur** [comment]
@repeatedly  Please review it as soon as possible. We need these changes in prod.

Thanks,
Sharad

**Pull request merged**